.github/project_workflows/add_device_profile.yml

```diff
@@ -0,0 +1,67 @@
+name: Test Add device and regenerate profiles with fastlane match
+
+# SECRETS needed:
+### SSH_PRIVATE_KEY for the match Repo
+### MATCH_PASSWORD
+### APPSTORE_CONNECT_API_KEY
+### API_KEY_ID
+### ISSUER_ID
+
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      devices:
+        description: 'JSON of devices. `{"Luka iPhone 6": "1234567890123456789012345678901234567890","Felix iPad Air 2": "abcdefghijklmnopqrstvuwxyzabcdefghijklmn"}`'
+        required: true
+        type: string
+      platform:
+        description: 'The platform'
+        required: true
+        default: 'ios'
+        type: choice
+        options:
+        - ios
+        - mac
+
+jobs:
+  create_files:
+    name: Create certificates and profiles
+    runs-on: macOS-latest
+    steps:
+    - name: Checkout the repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Install SSH key
+      uses: webfactory/ssh-agent@v0.5.4
+      with:
+        ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
+
+    - name: Cache Pods
+      uses: actions/cache@v3
+      id: cocoapodCache
+      with:
+        path: Pods
+        key: ${{ runner.os }}-pods-${{ hashFiles('**/Podfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-pods-
```

Comment:
I don't think we need to cache the pod here.

Reply:
@phongvhd93 updated at [c696076](https://github.com/nimblehq/ios-templates/pull/551/commits/c69607606f868ae1423df8160a43832e67bfb451)